index.html

```diff
@@ -16,7 +16,67 @@
 </head>
 <body>
   <header>
+
+    <div class="header-div">
+      <div class="head-box-text">Code Journal </div>
```

Comment:
If you put this text into a div with the `container` class, it will be at the same level of indentation as the main content without the need for an additional class. Ideally, you would also have a `row` class element as a child, with a `column-full` element child for that row. Inside that `column-full` class element, you should put the text in the header into an h1-h6 element